python/apps/taiga/src/taiga/stories/stories/notifications/__init__.py

```diff
@@ -31,14 +33,37 @@ async def notify_when_story_status_change(story: Story, status: str, emitted_by:
         emitted_by=emitted_by,
         notified_users=notified_users,
         content=StoryStatusChangeNotificationContent(
-            projects=story.project,
+            project=story.project,
             story=story,
             changed_by=emitted_by,
             status=status,
         ),
     )
 
 
+async def notify_when_story_workflow_change(story: Story, workflow: str, status: str, emitted_by: User) -> None:
+    """
+    Emit notification when a story workflow changes
+    """
+    notified_users = {u async for u in story.assignees.all()}
```

Comment:
I'm not sure this should go in the service layer, but I've seen it previously in other methods, so it's ok.